ui/pages/Search/components/PageHeader.jsx

```diff
@@ -19,7 +19,14 @@ const PAGE_CONFIGS = {
     entity: analysisGroupsByGuid[entityGuid],
     entityUrlPath: `analysis_group/${entityGuid}`,
   }),
-  families: entityGuid => ({ description: `Searching in ${entityGuid.split(/[,:]/).length} Families` }),
+  families: (entityGuid, p, f, a, s, searchFamiliesByHash) => {
```

Comment:
what do the params `p, f, a, s` represent?

Reply:
`projectsByGuid, familiesByGuid, analysisGroupsByGuid, searchesByHash`